P5/Source/Guidelines/en/CMC-ComputerMediatedCommunication.xml

```diff
@@ -0,0 +1,1326 @@
+        virtual avatar of an interlocutor in a 3D environment,</item>
+      <label rend="bold">onscreen activities,</label>
+      <item>i.e. non-bodily expressions that are transmitted to the group of interacting or
+        coworking participants, for instance the editing of content in a shared text editor which
+        can be perceived by the other parties simultaneously (as may be the case in learning or
+        collaboration environments).</item>
+    </list>
+    <p>The right part of <ref target="#cmcunits-taxonomy">the taxonomy</ref> describes units in
+      which the production, transmission, and perception of contributions to CMC interactions are
+      organized in a strictly consecutive order: The content—verbal, nonverbal, or multimodal—of
+      the contribution has to be produced before it can be transmitted through a network and made
+      available on the computer monitor or mobile screen of any other party as a preserved and
+      persistent unit. We term this type of unit a <term>post</term>. Posts occur in
+      two different variants: <list rend="bulleted">
+        <item>as <emph rend="bold">written or multimodal posts,</emph> which are produced with an
+          editor form that is designed for the composition of stretches of written text. Most
+          contemporary post-based CMC technologies provide features for the inclusion of graphic and
+          audiovisual content (emoji graphics, images, videos) into posts and even to produce posts
+          without verbal content (which then may consist only of emojis, an image, or a video file).
+          Written and multimodal posts are the standard formats for user contributions in primarily
+          text-based CMC genres and applications such as chat, SMS, WhatsApp, Instagram, Facebook, X
+          (Twitter), online forums, or Wikipedia talk pages.</item>
+        <item>as <emph rend="bold">audio posts</emph>, which are produced using a recording
+          function. In contrast to CMC units of the type <term>utterance</term> which
+          are produced and transmitted simultaneously, audio posts first have to be recorded as a
+          whole and are then transmitted, as audio files, via the internet; the availability of
+          the recording is indicated in the screen protocol by a template-generated, visual post;
+          the recipients can play the recording (repeatedly) by activating the play button displayed
+          in the post on the screen. Examples of CMC applications that implement audio posts are
+          WhatsApp or RocketChat.</item>
+      </list></p>
+    <p>Three of the four basic CMC units described above can be represented with models that are
+      described elsewhere in the TEI Guidelines:</p>
+    <table>
+      <row rend="underline">
+        <cell>CMC unit</cell>
+        <cell>Type of corpus data</cell>
+        <cell>TEI P5 element</cell>
+      </row>
+      <row>
+        <cell>utterance</cell>
```

Comment:
further above the category is named spoken utterance. I would call it the same as above.